tests/CiIntegrationTest.php

```diff
@@ -80,39 +80,34 @@ public function tearDown()
         )));
     }
 
-    public function testIntegration()
+    public function testIntegrationWithRandomChanges()
     {
         static::executeCommand(implode(' && ', array(
-            'git checkout -b case1 -q',
+            'git checkout -b random-changes -q',
             'touch dir\ a/file.php',
             'rm -r dir\ c',
             'echo "" >> dir\ b/file\ b.php',
             'echo "echo 1;" >> dir\ b/file\ b.php',
             'git add .',
-            'git commit -m "case1" -q',
+            'git commit -m "Random changes" -q',
         )));
 
         $integrationScript = explode("\n", str_replace('vendor/bin/', './../../../', file_get_contents(__DIR__.'/../dev-tools/ci-integration.sh')));
         $steps = array(
-            'COMMIT_RANGE="master..case1"',
+            'COMMIT_RANGE="master..random-changes"',
             $integrationScript[3],
-            $integrationScript[4],
-            $integrationScript[5],
+            "$integrationScript[4]; $integrationScript[5] $integrationScript[6]; $integrationScript[7]",
```

Comment:
I did this for readability of the suggestion, but as it's meant for CI integration, it doesn't really matter so I joined the lines in the suggestion directly.